src/std/Makefile

```diff
@@ -6,8 +6,8 @@
 CXXFLAGS=-O3 -std=c++17 -stdpar -DSTD
 STD_CXX=nvc++
 
-std-stream: main.cpp STDStream.cpp
-	$(STD_CXX) $(CXXFLAGS) $^ $(EXTRA_FLAGS) -o $@
+std-stream: ../main.cpp STDStream.cpp
+	$(STD_CXX) $(CXXFLAGS) $^ $(EXTRA_FLAGS) -o $@ -I. -I..
```

Comment:
`-I.` shouldn't be required here as we're not including any local headers that aren't already included with `#include "name.h"`